stable/traefik/README.md

```diff
@@ -335,7 +335,7 @@ acme:
 
 #### Example: AWS Route 53
 
-Route 53 requires the [following configuration variables to be set](values.yaml#L98-L101):
+Route 53 requires the [following configuration variables to be set](https://github.com/helm/charts/blob/68da43ea17582b6c74d8b8956b1b20ee49d8d420/stable/traefik/values.yaml#L230-L232):
```

Comment:
```suggestion
Route 53 requires the [following configuration variables to be set](values.yaml#L230-L232):
```

Reply:
My issue with this suggestion is it's likely to lead to the same problem. The moment any other change is made to this document, the values in this suggestion will be irrelevant. At least this way, the link is to something that is valid, available, and provides some context as to what those configuration variables were at some point.

Reply:
Thanks for your feedback. 
So to me me it will preferable to don't reference lines.
Maybe we should remove the link to specific lines. There is already an example few lines below. 

https://github.com/helm/charts/pull/8415/files#diff-2cdc83a6a0baf42fe79824b104b24e2eR346

WDYT ?

Reply:
Removing the link would be a good move towards consistency, but still as a user, I found the link useful because it contained [the relevant configuration details for a number of DNS providers](https://github.com/helm/charts/blob/68da43ea17582b6c74d8b8956b1b20ee49d8d420/stable/traefik/values.yaml#L165-L234). Is there another place where these expectations are documented that we could link users to instead of anything contained within the repo (for the aforementioned reasons)?

Reply:
Of course, we can create a link to https://docs.traefik.io/configuration/acme/#provider

Reply:
This is perfect, I've made the relevant updates.

Reply:
This is perfect, I've made the relevant updates.